app/views/requests/edit.html.erb

```diff
@@ -6,6 +6,6 @@
   <% end %>
 
   <%= c 'section' do %>
-    <%= c 'request_form', organization: @organization, request: @request %>
+    <%= c 'request_form', organization: @organization, request: @request, available_tags: @available_tags %>
```

Comment:
Same here - and it was already like this. Shouldn't we just use request.organization internally and get rid of the parameter organization?